### PR TITLE
Change deprecated no-ssr to client-only

### PR DESCRIPTION
**Pull request by Yusadolat**
Since no-ssr is now deprecated, I changed it to client-only. Screenshot below!

client-only

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "vue-intersect": "^1.1.3",
     "vue-localstorage": "^0.6.2",
     "vue-media": "^1.1.1",
-    "vue-no-ssr": "^1.1.1",
+    "vue-client-only": "^1.1.1",
```
Comment (bovas85) [inline package.json]: Thanks for the PR! Can you remove this package as it doesn't exist? We don't need vue-no-ssr either as it's now built in into Nuxt

Comment (Yusadolat) [reply package.json]: Oh, my bad. I just did. Can you check now?

File: yarn.lock [hunk 2]
```diff
@@ -978,7 +978,7 @@
     vue "^2.6.10"
     vue-client-only "^2.0.0"
     vue-meta "^2.2.2"
-    vue-no-ssr "^1.1.1"
+    vue-client-only "^1.1.1"
```
Comment (bovas85) [inline yarn.lock]: You need to reset the lock file as it's still referencing this.

**Comment by bovas85** [comment]
Thanks, all good!

**Comment by Yusadolat** [comment]
Thank you!
You're the best! :+1:

**Pull request merged**